datasets/kwaliteitsmonitorwonen/dataset.json

```diff
@@ -0,0 +1,160 @@
+            "description": "Oorspronkelijke bouwjaar van het pand"
+          },
+          "pndLigging": {
+            "type": "string",
+            "description": "Ligging van het pand"
+          },
+          "mntMonumentstatus": {
+            "type": "string",
+            "description": "Monumentstatus van het pand"
+          },
+          "bsdLigtInBeschermdGebied": {
+            "type": "string",
+            "description": "Ligt het verblijfsobject in een beschermd gebied?"
+          },
+          "bsdBeschermdStadsdorpsgezicht": {
+            "type": "string",
+            "description": "Tot welk gebied van het beschermd stadsdorpsgezicht het verblijfsobject behoort"
+          },
+          "adres": {
+            "type": "string",
+            "description": "Adres van het verblijfsobject"
+          },
+          "huisnummer": {
+            "type": "integer",
+            "description": "Huisnummer van het verblijfsobject"
+          },
+          "huisletter": {
+            "type": "string",
+            "description": "Huisletter van het verblijfsobject"
+          },
+          "huisnummertoevoeging": {
+            "type": "string",
+            "description": "Huisnummertoevoeging van het verblijfsobject"
+          },
+          "postcode": {
+            "type": "string",
+            "description": "Postcode van het verblijfsobject"
+          },
+          "woonplaats": {
+            "type": "string",
+            "description": "Woonplaats van het verblijfsobject"
+          },
+          "bouwblokCode": {
+            "type": "string",
+            "description": "Code van het bouwblok van het verblijfsobject"
+          },
+          "buurtNaam": {
+            "type": "string",
+            "description": "Naam van de buurt van het verblijfsobject"
+          },
+          "buurtCode": {
+            "type": "string",
+            "description": "Code van de buurt van het verblijfsobject"
+          },
+          "wijkNaam": {
+            "type": "string",
+            "description": "Naam van de wijk van het verblijfsobject"
+          },
+          "wijkCode": {
+            "type": "string",
+            "description": "Code van de wijk van het verblijfsobject"
+          },
+          "stadsdeelNaam": {
+            "type": "string",
+            "description": "Naam van het stadsdeel van het verblijfsobject"
+          },
+          "stadsdeelCode": {
+            "type": "string",
+            "description": "Code van het stadsdeel van het verblijfsobject"
+          },
+          "gebiedsgerichtWerkenNaam": {
+            "type": "string",
+            "description": "Naam van het gebiedsgericht werken"
+          },
+          "gebiedsgerichtWerkenCode": {
+            "type": "string",
+            "description": "Code van het gebiedsgericht werken"
+          },
+          "vveIsEigendomVve": {
+            "type": "string",
+            "description": "Is het verblijfsobject eigendom van de VVE?"
```

Comment:
Ook hier: moet dit niet een boolean zijn, of gewoon een `vveStatutaireNaam` van null in geval van "nee"?

Reply:
Dit is ook een string